### DISCUSSION TITLE
Can't join current battle #28 Revert?

### DISCUSSION THREAD

**Discussion by nynexman4464** [cssbattle/forum#360] [Bugs reports]
Maybe I'm confused about how battles work, but it seems like this battle has started, but I can't join it.  The first two entries just say "Unlocks in" with no numbers.  The others have a countdown (EG 6 days 23 hours).  Am I just confused on how to join, or is something broken?  I was able to submit something for a past battle as well as one of the daily targets.

![Screenshot 2023-10-12 at 1 31 42 PM](https://github.com/cssbattle/forum/assets/1675576/0a008e02-0ee6-4706-8fe7-f5a6552ae7b3)

![Screenshot 2023-10-12 at 1 25 42 PM](https://github.com/cssbattle/forum/assets/1675576/63a0f76f-d79f-40c0-b89f-1895512b962c)

**Comment by tendev2** [comment]
i have the same problem,maybe they have a technical essue!

**Comment by chinchang** [comment]
We have encountered a technical issue in our battle deployment. The battle is tentatively postponed to tomorrow. https://x.com/css_battle/status/1712527808357085384?s=20

**Reply by nynexman4464** [reply]
ah ok, thanks
